sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample1_DetectEntireSeriesAnomaly.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Azure;
+using Azure.AI.AnomalyDetector;
+using Azure.AI.AnomalyDetector.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+class Sample1_DetectEntireSeriesAnomaly
```

Comment:
We run the samples as a part of tests. Could you please refer [these samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples) and change `Sample1_DetectEntireSeriesAnomaly`, `Sample2_DetectLastPointAnomaly` and `Sample3_DetectChangePoint` to something like [this](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample1_RecognizeContentFromUri.cs)?

Reply:
For this you would need to move samples folder inside [tests](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/anomalydetector/Azure.AI.AnomalyDetector/tests).

Reply:
Done.

---
In reply to: [481444328](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r481444328) [](ancestors = 481444328)

Reply:
Done.

---
In reply to: [481450329](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r481450329) [](ancestors = 481450329)